canonical-web-and-design/maas-ui
# refactor(ui): refactor cache sets table to use new utils

## Done

- Refactored cache sets table to no longer use the `separateStorageData` util.
- Removed old utils file and replaced with new one

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the storage tab of a machine that has cache sets (or go to a ready/allocated machine in he angular version and add a cache set from the available disks table)
- Check that the data is the same as before

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2281
